iModelCore/iModelPlatform/DgnCore/DgnDbSchema.cpp

```diff
@@ -557,7 +566,7 @@ BeSQLite::DbResult DgnDb::_OnBeforeProfileUpgrade()
 /*---------------------------------------------------------------------------------**//**
 * @bsimethod
 +---------------+---------------+---------------+---------------+---------------+------*/
-BeSQLite::DbResult DgnDb::_OnAfterProfileUpgrade()
+BeSQLite::DbResult DgnDb::_OnAfterProfileUpgrade(Db::OpenParams const& params)

```

Comment:
you don't use params - why bother passing it?

Reply:
fixed